### ISSUE TITLE
Guake first run via F12 does not take focus

### ISSUE THREAD

**Issue by assmblyreq** [Guake/guake#1861]
**Describe the bug**

On first boot, Guake is autorun. First press F12 hotkey, Guake appears, but does not have window focus or keyboard input.  To have window focus, F12 must be pressed 2 more times to make Guake hide and reappear.  Subsequent used of Guake apeear as expected even after logging out of Guake via Ctrl-D or exit command.  This issue is only on the very first appearance of Guake to a new Gnome session.

<details><summary>$ guake --support</summary>

Guake Version:		3.6.3

Vte Version:		0.62.3

Vte Runtime Version:	0.62.3

--------------------------------------------------
GTK+ Version:		3.24.24

GDK Backend:		GdkX11.X11Display

--------------------------------------------------
Desktop Session: gnome-xorg

--------------------------------------------------
Display: :1

RGBA visual: True

Composited: True

* Monitor: 0 - SAM DVI-D-0
    * Geometry:		1920 x 1080 at 0, 0
    * Size:		521 x 293 mm²
    * Primary:		True
    * Refresh rate:	60.00 Hz
    * Subpixel layout:	unknown

**Comment by Davidy22** [comment]
I finally restarted my machine to test this and did not encounter this bug. Does this still happen on the latest build for you?

**Comment by assmblyreq** [comment]
Yes.

**Comment by pabloab** [comment]
Me too. Also, the window does not span through the screen (if not related I could create a new issue)

![image](https://user-images.githubusercontent.com/657836/143992425-29de0bdb-464f-4d0e-83d8-9a9ea4729adc.png).

The window issue always happened/happens when Preference window (`guake -p`) is open.

```
Guake Terminal: 3.7.0
VTE: 0.60.3
VTE runtime: 0.60.3
Gtk: 3.24.20
```

**Comment by ckjoris** [comment]
I have the same issue with needing to hit the keyshortcut twice, first to show guake, then again to focus it.
However for me the issue happens not only after boot, but every time. Maybe its because I'm on gnome+wayland.

Managed to make a workaround with a custom toggle-guake command https://gist.github.com/ckjoris/cf2b3a7ef729e629409db47d29009b04

**Comment by assmblyreq** [comment]
I solved it by uninstalling Guake and using the ddterm extension instead

**Comment by pabloab** [comment]
[ddterm](https://github.com/ddterm/gnome-shell-extension-ddterm) only works with GNOME Terminal, right?

I'm using [quake-mode](https://github.com/repsac-by/gnome-shell-extension-quake-mode), with Alacritty in one host and GNOME Terminal in other.

**Comment by Aliw7979** [comment]
Didn't you get a fix yet?

**Comment by assmblyreq** [comment]
Oh, gracious, yes! But the solution was to uninstall Guake and use a Gnome
extension drop-down terminal. I have no idea if Guake got patched.
